build-system/tasks/get-zindex/index.js

```diff
@@ -182,16 +184,14 @@ function getZindexChainsInJs(glob, cwd = '.') {
 
       try {
         const reportParsed = JSON.parse(report);
-
         if (reportParsed.length) {
           result[relative] = reportParsed.sort(sortedByEntryKey);
         }
+        if (--resultCountInverse === 0) {
+          resolve(result);
+        }
```

Comment:
Does it _always_ reach this, even if the sub-process returns a failure? If so, LGTM - otherwise need a way to determine when a sub-process failed and count that too

Reply:
I believe this error handler would end the process. 

https://github.com/ampproject/amphtml/blob/747bf4387e5ea6955df09d24cd2da4d476456d92/build-system/tasks/get-zindex/index.js?plain=1#L167-L169

Reply:
Works for me!